recipes/dvc/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "dvc" %}
+{% set version = "0.53.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 31d92fc583351664ca48df146e115c66ab41070520bb02ef35e14a9dec156ea5
+
+build:
+  skip: true  # [py<27 or py30 or py31 or py32 or py33 or py34]
```

Comment:
Please remove the skip and add `- python >=3.5` to the host and run requirements instead. This can be then also  `noarch: python` again.

Reply:
Why, that skip list? conda-forge anyhow currently only supports  python 2.7, 3.6, 3.7.

Reply:
@dbast oh, thanks! it makes sense to remove it completely then.